sympy/printing/c.py

```diff
@@ -88,6 +89,13 @@
 
 reserved_words_c99 = ['inline', 'restrict']
 
+
+def _is_valid_c_variable_name(var_name):
```

Comment:
Could we make this a method (perhaps a static method). So that a user can override this easily in a custom printer inheriting from CCodePrinter?